app/Http/Controllers/Three/PostsController.php

```diff
@@ -101,14 +102,13 @@ public function store(PostRequest $request)
     {
         $transactionId = incrementTransactionId($request);
 
-        $signup = $this->signups->get($request['northstar_id'], $request['campaign_id'], $request['campaign_run_id']);
+        $signup = $this->signups->get(auth()->id(), $request['campaign_id'], $request['campaign_run_id']);
```

Comment:
@DFurnes should this also be updated to check if a `northstar_id` is sent in the request, and if so, send that instead of getting it from `auth()->id()`?